### DISCUSSION TITLE
Slow blackarch download

### DISCUSSION THREAD

**Discussion by tonkaxxx** [BlackArch/blackarch#4399] [Support]
Type: Question

## Question
when im trying to download blackarch tools (with "sudo pacman -S blackarch") , it happens, but rlly slow. I have fresh arch and good internet

**Comment by D3vil0p3r** [comment]
Please don't install the entire `blackarch` category because there is no sense. You won't use even 1% of installed tools. 
* Can you perceive it as slow because the download size is huge?
* If not, could be your Internet connection experienced performance degradation?
* Still, If not, edit `/etc/pacman.d/blackarch-mirrorlist` comment all lines except the ones related to your country or a nearer one. Save the file and run `sudo pacman -Syy` and try to install the tools YOU NEED and not the entire BlackArch category.

If you solve, please close the issue.

**Comment by noraj** [comment]
https://miloserdov.org/?p=6927
